pootle/core/markup/filters.py

```diff
@@ -133,7 +133,23 @@ def apply_markup_filter(text):
         elif markup_filter_name == 'markdown':
             import bleach
             import markdown
-            html = bleach.clean(markdown.markdown(text, **markup_kwargs))
+
+            # bleach.ALLOWED_TAGS is currently: [
+            #   u'a', u'abbr', u'acronym', u'b', u'blockquote', u'code', u'em',
+            #   u'i', u'li', u'ol', u'strong', u'ul']
+            MARKDOWN_DEFAULT_TAGS = bleach.ALLOWED_TAGS + [
+                u'h1', u'h2', u'h3', u'h4', u'h5',
+                u'p', u'pre',
+                u'img',
+                u'hr',
+            ]
+
+            tags = MARKDOWN_DEFAULT_TAGS
+            if 'clean' in markup_kwargs:
+                if 'allowed_tags' in markup_kwargs['clean']:
+                    tags = markup_kwargs['clean']['allowed_tags']
+            html = bleach.clean(markdown.markdown(text, **markup_kwargs),
```

Comment:
No, I specifically wanted someone to be on their own if they want to control the allowed tags.  Otherwise there is no way for them to fully override things.